apps/src/applab/designProperties.jsx

```diff
@@ -47,16 +59,113 @@ var DesignProperties = module.exports = React.createClass({
         handleDelete={this.props.onDelete}/>);
     }
 
+    var tabHeight = 35;
+    var borderColor = '#c6cacd';
+    var bgColor = '#e7e8ea';
+
+    var defaultTabStyle = {
+      borderColor: borderColor,
+      borderStyle: 'solid',
+      borderWidth: '1px 1px 1px 0',
```

Comment:
bootstrap does have a tab control which you could possibly use here?  we've used it in at least a couple other places, such as code.org/learn.

your trick is fun, but it might be worth commenting for future people curious about what the specific work does.  i don't know whether there's a better way, but this does seem to work.  

i do find the borderWidth four parameters a bit confusing sometimes, and wonder if breaking it up into borderWidth-top & borderWidth-left or whatever they are might make it a bit more readable.